app-misc/pet/pet-0.3.0.ebuild

```diff
@@ -4,19 +4,21 @@
 EAPI=6
 
 EGO_VENDOR=( "github.com/BurntSushi/toml b26d9c308763d68093482582cea63d69be07a0f0"
```

Comment:
Are you really fixing dependencies or did the dependencies require updating as part of bump? If the latter, then please merge it into the original commit doing the bump. It's good to split other changes into additional commits but changes that are necessary for the ebuild to work in the first place belong in the first commit.

Reply:
Upstream changed dependencies - I am just following him in ebuild.
It always a good idea to have dependencies in upstream to be consistent with dependencies in ebuild. So, yes, dependencies require updating as a part of bump.